roles/edpm_reboot/molecule/default/molecule.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
for molecule tests like this one, which require at least a vm to be executed, take inspiration from `edpm_nova` or `edpm_libvirt` molecule tests.

Reply:
In fact it requires two vms(two node set) to be executed. 
With ansible_connection set to local reboot won't be performed:

```

TASK [osp.edpm.edpm_reboot : Reboot nodes] *************************************
fatal: [compute-1]: FAILED! => changed=false
  elapsed: 0
  msg: Running ansible.builtin.reboot with local connection would reboot the control node.
  rebooted: false

PLAY RECAP *********************************************************************
```

Reply:
I changed test driver back to podman. As I mentioned in my previous comment to test running reboot a zuul job configuration would require to be expanded to at least two vms. One vm would need to be ansible controller and other ansible target. I think this kind of test could be ran in some upper test layer.